projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.scss

```diff
@@ -33,7 +24,7 @@ geoengine-color-attribute-input {
     grid-template-rows: 1fr 1fr;
     grid-gap: 5px;
     height: 5rem;
-    width: 100%;
+    width: 95%;
```

Comment:
I wrote it as `calc(100% - 0.5rem)` to make sure the control elements fill their row otherwise, is that OK?
[7fbcc6e](https://github.com/geo-engine/geoengine-ui/pull/614/commits/7fbcc6edc42516577798dfccc44d9fedc99f57fd)